CakePHP/Sniffs/Commenting/FunctionCommentTypeSniff.php

```diff
@@ -1,96 +0,0 @@
-<?php
-/**
- * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
- * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- *
- * Licensed under The MIT License
- * For full copyright and license information, please see the LICENSE.txt
- * Redistributions of files must retain the above copyright notice.
- *
- * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- * @link          http://pear.php.net/package/PHP_CodeSniffer_CakePHP
- * @license       http://www.opensource.org/licenses/mit-license.php MIT License
- */
-
-/**
- * Asserts that function comments use the short form for types:
- * - bool instead of boolean
- * - int instead of integer
- */
-class CakePHP_Sniffs_Commenting_FunctionCommentTypeSniff implements PHP_CodeSniffer_Sniff
```

Comment:
Once again I might be missing something but I'm pretty sure that FunctionCommentSniff.php does the same jobs. You can check in the tests for the other Sniff,  the parameters are correctly detected as wrong.